src/routes.js

```diff
@@ -1,9 +1,22 @@
 import React from 'react';
-import { Route } from 'react-router';
-import { HelloWord } from 'views/HelloWord';
+import { Switch, Route } from 'react-router-dom';
+import HelloWorld from 'views/HelloWorld';
 
-export default (
-    <Route>
-        <Route path="/" component={HelloWord} />
-    </Route>
-);
+// This is a class-based component because the current
+// version of hot reloading won't hot reload a stateless
+// component at the top-level.
+/* eslint-disable react/prefer-stateless-function */
+
+class App extends React.Component {
+    render() {
+        return (
+            <div>
+                <Switch>
+                    <Route exact path="/" component={HelloWorld} />
+                </Switch>
+            </div>
+        );
+    }
+}
+
+export default App;
```

Comment:
I would change this line to `export default Routes` and would rename the class as well, because you are in a route file, and you import this as `import Routes from './routes';`, so I think that is the best way to follow good patterns.